### ISSUE TITLE
AVT 3 NumberInput  Components - VO does not announce value or increment when the up and down arrows used to input the number

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4185]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
With VoiceOver enabled Tab to number input. (Focus is on the up arrow)
Press the up arrow. Voice over does not announce the value or the increment number.
Tab to the down arrow is the same.
Tab to the Number input and the value is announced.
Enter an odd number directly into the Number field.
Shift-Control-Option to enter the stepper
Control Option Up or Down Arrow keys to change the number.
The number is not announced correctly by VoiceOver.

## Expected result:
When on an arrow that increments the number up or down the user should be advised that the number is incrementing by 10 as well as the number that is displayed in the number input field.
If a user inputs a number directly that is in the wrong increments they should get a warning.
If they change the number using the VO Control-Option-Up or Down arrow keys the number should be announced correctly.

**Comment by abbeyhrt** [comment]
@snidersd if you get the chance, could you try out VO in Safari for the NumberInput? I'm finding that `aria-atomic`, which should help the value announce correctly when using arrow keys,  just isn't working in Chrome but it is in Safari and Firefox.

**Comment by snidersd** [comment]
@abbeyhrt - I just tested this with macOS on Safari 13.0.2 and Win10 on FF 68.2.0. The only issue found is that the arrows that increment the number up and down are in the Tab order, but the number is not announced when it changes using the keyboard. I also tested it on macOS with FF 68 (ESR) and Chrome 78.0.3904.70 and both fail.

**Comment by dakahn** [comment]
bug filed in Chromium bug tracker [here](https://bugs.chromium.org/p/chromium/issues/detail?id=1043216&can=2&q=aria-atomic)

**Comment by dakahn** [comment]
Looks like the Chromium team was unable to reproduce the error. Unblocking and moving to backlog. 
https://bugs.chromium.org/p/chromium/issues/detail?id=1043216#c4

**Comment by emyarod** [comment]
since the native HTML number input and w3's spin buttons don't allow keyboard navigation to the increment and decrement buttons, would it be better to just use arrow keys while the number input field is focused rather than tabbing to the increment/decrement buttons? I think we could get VO to announce changes caused by the increment/decrement buttons with `aria-live` but that may result in redundant readings

**Comment by emyarod** [comment]
as it stands now though, arrow keys while the input field is focused will cause VO to correctly announce the value

![image](https://user-images.githubusercontent.com/8265238/110987299-7f2b1500-8334-11eb-9d8c-4abf627cd0bc.png)

arrow key stepping seems to read correctly in VO

on a side note #8090 will remove the nonnegative tabindex on the input controls so that it matches the native number input and the w3 examples
